src/mixins/nft.js

```diff
@@ -256,39 +261,33 @@ export default {
     },
     async updateNFTHistory() {
       this.isHistoryInfoLoading = true;
-      let hasHistoryInDatabase = false;
+      const historyOnChain = await this.getNFTEventsAll();
+      let history = historyOnChain;
```

Comment:
Not sure if you want to clone this in case some one modify `history` but accidentally modify `historyOnChain`

Reply:
`history` and `historyOnChain` only live in `updateNFTHistory()` method. If we assign `history` to `this.NFTHistory` (`this.NFTHistory = history;`) after all the process, can anyone still be able to modify it from somewhere outside the method?

Reply:
if it only live in this function i guess it is ok, but later if this function expands maybe someone will mess this up